tests/patches/glpi-allow-test-on-master-branch.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.php b/setup.php
+index eed6660..1f065eb 100644
+--- a/setup.php
++++ b/setup.php
+@@ -209,7 +209,7 @@ function plugin_version_flyvemdm() {
+       'requirements'   => [
+          'glpi' => [
+             'min' => $glpiVersion,
+-            'max' => PLUGIN_FLYVEMDM_GLPI_MAX_VERSION,
++            'max' => '9.5',
```

Comment:
Funny you did not changed the constant itself :D This way, when GLPI 9.5 will begin, we may simply create a PR with the patch